src/status_im2/navigation/events.cljs

```diff
@@ -29,11 +34,21 @@
    :dispatch      [:hide-bottom-sheet]
    :open-modal-fx comp})
 
+(rf/defn dismiss-modal
+  {:events [:dismiss-modal]}
```

Comment:
how this will work? i don't see its usage, it should be used in `navigate-back-within-stack` no?

Reply:
It's a bit redundant, as as you say, `navigate-back-within-stack` would dismiss the modal if is the last screen in the stack. We could remove it I suppose.

Reply:
But it may be useful for cases where we want to dismiss a modal from a screen in the stack different from the first one in the modal stack.